frameworks/g-cloud-12/messages/urls.yml

```diff
@@ -0,0 +1,9 @@
+framework_agreement_url: "https://www.gov.uk/government/publications/g-cloud-12-framework-agreement"
+
+framework_agreement_pdf_url: "https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/810459/g-cloud-12-framework-agreement.pdf"
+
+supplier_guide_url: "https://www.gov.uk/guidance/g-cloud-suppliers-guide#how-to-apply"
+
+call_off_contract_url: "https://www.gov.uk/government/publications/g-cloud-12-call-off-contract"
+
+customer_benefits_record_form_url: "https://forms.gle/ir3a77A5knkUuiJ38"
```

Comment:
I might tweak the script to put proper placeholders in for these - `/replace/with/path/to/published/document` etc so it's clear that they need review.

Reply:
👍 
Put in __placeholder__ tags for these as a reminder